### PR TITLE
[ReplayKit] Updates for Xcode13 Beta 1

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/replaykit.cs [hunk 1]
```diff
@@ -136,6 +136,21 @@ interface RPScreenRecorder {
 		[Async]
 		[Export ("stopRecordingWithOutputURL:completionHandler:")]
 		void StopRecording (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
+
+		[Async]
+		[NoTV, NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("exportClipToURL:duration:completionHandler:")]
+		void ExportClip (NSUrl url, double duration, [NullAllowed] Action<NSError> completionHandler);
```
Comment (tj-devel709) [inline src/replaykit.cs:143]: I'm hesitant about changing this ones name from ExportClipToUrl to ExportClip. 
NSUrl is the first argument, but does this name change make it more confusing to use?

File: src/ReplayKit/RPEnums.cs [hunk 2]
```diff
@@ -53,6 +53,7 @@ public enum RPRecordingError : long {
 		FailedToStartCaptureStack = -5833,
 		InvalidParameter = -5834,
 		FilePermissions = -5835,
+		ExportClipToURLInProgress = -5836,
```
Comment (rolfbjarne) [inline src/ReplayKit/RPEnums.cs:56]:
```suggestion
		ExportClipToUrlInProgress = -5836,
```

File: src/replaykit.cs (hunk 1 above)
Comment (rolfbjarne) [reply src/replaykit.cs:143]: When in doubt, you can check the Swift naming, in this case it's `exportClip` (https://developer.apple.com/documentation/replaykit/rpscreenrecorder/3752130-exportclip), so I think this is totally fine.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

**Review by mandel-macaque — APPROVED** [review]

File: src/replaykit.cs (hunk 1 above)
Comment (tj-devel709) [reply src/replaykit.cs:143]: Makes sense! Thanks!

**Review by rolfbjarne — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**